analysis/wwz/wwz4l.py

```diff
@@ -217,6 +223,8 @@ def process(self, events):
         mu   = events.Muon
         tau  = events.Tau
         jets = events.Jet
+        if ("2022" in year) and (not isData):
```

Comment:
But events.Pilepu is used for non 2022 years too, right? Is there a particular reason this if is only applied for 2022?

Also, it looks like `pileup` is only used in a few limited places. Would it be easier to just use `events.Pileup` there, as we do not R2?